rasa/shared/core/domain.py

```diff
@@ -1045,12 +1047,39 @@ def as_dict(self) -> Dict[Text, Any]:
             KEY_INTENTS: self._transform_intents_for_file(),
             KEY_ENTITIES: self._transform_entities_for_file(),
             KEY_SLOTS: self._slot_definitions(),
-            KEY_RESPONSES: self.templates,
+            KEY_RESPONSES: self.get_responses_with_multilines(self.templates),
             KEY_ACTIONS: self._custom_actions,  # class names of the actions
             KEY_FORMS: self.forms,
             KEY_E2E_ACTIONS: self.action_texts,
         }
 
+    @staticmethod
+    def get_responses_with_multilines(
+        responses: Dict[Text, List[Dict[Text, Any]]]
+    ) -> Dict[Text, List[Dict[Text, Any]]]:
+        """Return `responses` with preserved multilines in the `text` key.
```

Comment:
```suggestion
        """Returns `responses` with preserved multilines in the `text` key.
```

Reply:
I need to make sure to read our style guide  every two weeks 🤦